src/GitGlyph/GitBlameVisitor.cs

```diff
@@ -23,7 +23,7 @@ public GitBlameVisitor(Repository r, ILog logger)
         public string Visit(string text, IReadOnlyDictionary<string, string> context)
         {
             var path = System.IO.Path.GetFullPath(context[ContextKeys.FilePath]);
-            var blame = GetBlame(path)?.FirstOrDefault(bh => bh.FinalStartLineNumber.ToString() == context[ContextKeys.LineNumber]);
+            var blame = GetBlame(path)?.FirstOrDefault(bh => bh.FinalStartLineNumber == Int32.Parse(context[ContextKeys.LineNumber]) - 1);

```

Comment:
While you're here, I'd extract `Int32.Parse(context[ContextKeys.LineNumber]) - 1` into a separate local variable to make this line less complicated. Also not sure if perhaps you need to maybe use TryParse?